lib/crowi/index.js

```diff
@@ -136,11 +136,12 @@ Crowi.prototype.setupConfigPubSub = function() {
     debug('PubSubId', this.pubSubId)
 
     if (this.subscriber) {
-      this.subscriber.on('message', (channel, message) => {
+      this.subscriber.on('message', async (channel, message) => {
         if (channel !== this.configPubSubChannel) return
         const { pubSubId, config } = JSON.parse(message)
         if (pubSubId === this.pubSubId) return
         this.config = config
+        await Promise.all([this.setupSlack(), this.setupMailer()])
```

Comment:
[note] I thought that it's bit difficult to detect what instances to reload by difference from current config. Also it will confuse someone reading this.
This is why of adding this only.

Reply:
Also I checked that there are no problem to call `setupSlack` and `setupMailer` always.